roles/openshift_logging_curator/vars/default_images.yml

```diff
@@ -1,3 +1,3 @@
 ---
 __openshift_logging_curator_image_prefix: "{{ openshift_logging_image_prefix | default('docker.io/openshift/origin-') }}"
-__openshift_logging_curator_image_version: "{{ openshift_logging_image_version | default(openshift_image_tag) }}"
+__openshift_logging_curator_image_version: "{{ openshift_logging_image_version | default(openshift_image_tag[0:4]) }}"
```

Comment:
This will fail on 'v3.10' as that requries 5 characters.

Reply:
Good catch, updated to use regex instead